filebrowser/namers.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import unicode_literals
 import re
-from django.utils import six
+import six
```

Comment:
This import six...

Reply:
I've been trying to separate out built in imports, django imports, third party imports and local imports into sections with new lines. I think I've missed this one.